docs/docs/recipes.md

```diff
@@ -1655,50 +1655,50 @@ This is an **escape route** for [specific use cases](/docs/static-folder/#when-t
 │       └── fiesta.jpg
 ```
 
-2. In `index.js`, add an `<img>` tag with the `src` as the relative path of the file from the `static` folder, and add an `alt` attribute [describing the image](https://webaim.org/techniques/alttext/):
+2. No `index.js`, adicione uma tag `<img>` com o `src` como o caminho relativo do arquivo da pasta `static`, e adicione um atributo `alt` [descrevendo a imagem](https://webaim.org/techniques/alttext/):
 
 ```jsx:title=src/pages/index.js
-import React from "react"
+import React from 'react'
 
 export default () => (
-  <img src={`fiesta.jpg`} alt="A dog smiling in a party hat" /> // highlight-line
+  <img src={`fiesta.jpg`} alt='A dog smiling in a party hat' /> // highlight-line
 )
 ```
 
-3. Run `gatsby develop` to start the development server.
-4. View your image in the browser: `http://localhost:8000/`
+3. Execute `gatsby develop` para iniciar o servidor de desenvolvimento.
+4. Veja sua imagem no navegador:`http://localhost:8000/`
 
-#### Additional resources
+#### Recursos adicionais
 
-- [Example repo referencing an image from the static folder](https://github.com/gatsbyjs/gatsby/tree/master/examples/recipe-static-image)
-- [Using the Static Folder](/docs/static-folder/)
-- [More on all image techniques in Gatsby](/docs/images-and-files/)
+- [Exemplo de repositório referenciando uma imagem da pasta estática](https://github.com/gatsbyjs/gatsby/tree/master/examples/recipe-static-image)
+- [Usando a Pasta Static](/docs/static-folder/)
+- [Mais sobre todas as técnicas de imagem em Gatsby](/docs/images-and-files/)
 
-### Optimizing and querying local images with gatsby-image
+### Otimizando e consultando imagens locais com gatsby-image
 
-The `gatsby-image` plugin can relieve much of the pain associated with optimizing images in your site.
+O plugin `gatsby-image` pode aliviar grande parte da dor associada à otimização de imagens no seu site.
 
-Gatsby will generate optimized resources which can be queried with GraphQL and passed into Gatsby's image component. This takes care of the heavy lifting including creating several image sizes and loading them at the right time.
+O Gatsby irá gerar recursos otimizados que podem ser consultados com o GraphQL e passados ​​para o componente de imagem do Gatsby. Isso cuida do trabalho pesado, incluindo a criação de vários tamanhos de imagem e o carregamento no momento certo.
```

Comment:
Sugiro não usar tradução literal nesse trecho do heavy-lifting, pois a leitura nao fica tao natural. Exemplo de tradução não literal: 'takes care' poderia virar 'elimina' e ' of the heavy lifting' poderia virar ' da parte mais complexa'.